update studio 3.6.0 (#1902)
vesoft-inc/nebula-docs: docs-2.0/nebula-explorer/deploy-connect/ex-ug-connect.md

    !!! note

>         - When NebulaGraph and Explorer are deployed on the same machine, you must enter the IP address of the machine, but not `127.0.0.1` or `localhost`.

Reviewer: ```suggestion
        - When NebulaGraph and Explorer are deployed on the same machine, you must enter the IP address of the machine, instead of `127.0.0.1` or `localhost`.
```
